babel.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = function (api) {
     plugins: [
       '@babel/plugin-transform-runtime',
       '@babel/plugin-proposal-class-properties',
+      '@babel/plugin-proposal-export-default-from',
```

Comment:
Why did you decide to add this plugin? The syntax is very nice, but it is stage 1 and doesn't seem to be required.

Reply:
I did not know it was in Stage 1. I'll drop this and adjust the lint rules.